sdcm/stress_thread.py

```diff
@@ -128,11 +128,21 @@ def create_stress_cmd(self, node, loader_idx, keyspace_idx):
                           3]  # make sure each loader is targeting on datacenter/region
             first_node = first_node[0] if first_node else self.node_list[0]
             stress_cmd += " -node {}".format(first_node.ip_address)
-
-        stress_cmd += ' -errors skip-unsupported-columns'
-
+        stress_cmd = self._add_errors_option(stress_cmd, ['skip-unsupported-columns'])
         return stress_cmd
 
+    @staticmethod
+    def _add_errors_option(stress_cmd: str, to_add: list):
+        errors_option = [opt for opt in stress_cmd.split(' -') if opt.startswith('errors')]
```

Comment:
sounds like a perfect usecase for `regex.sub`

Reply:
fixed to re.sub